gradle/libs.versions.toml

```diff
@@ -1,6 +1,6 @@
 [versions]
 checker = "3.22.2"
-ideProbe = "0.31.0+3-0b91d467-SNAPSHOT"
+ideProbe = "0.31.0+7-5855d66c-SNAPSHOT"
```

Comment:
Soon to be bumped to 0.35.0 once https://github.com/VirtusLab/ide-probe/actions/runs/2614969110 completes (+ some Maven Central delays)

Reply:
Action done, now let's wait for https://repo1.maven.org/maven2/org/virtuslab/ideprobe/junit-driver_2.13/ to include 0.35.0